src/components/SelectGameSettings.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, InputLabel, MenuItem, Paper, Select, styled, Typography, useMediaQuery, useTheme } from '@mui/material';
+import Image from 'mui-image';
+import type { SelectChangeEvent } from '@mui/material/Select';
+import type { ButtonGroupButton } from './Buttons';
+import { ButtonGroup } from './Buttons';
+import type { ControlCallbacks, PageControl } from './controls';
+import { parseSpecialText } from '../functions-and-types';
+import type { GameSettings } from '../Projects';
+import Pages from '../Pages';
+
+export const SettingsCard = styled(Paper)(({ theme }) => ({
+    ...theme.typography.body2,
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+    borderColor: theme.palette.primary.light,
+    lineHeight: 2,
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+    paddingLeft: theme.spacing(0.5),
+    paddingRight: theme.spacing(0.5),
+}));
+
+/**
+ * Start page
+ */
+export function SelectGameSettings(props: SelectGameSettingsProps) {
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down('sm'));
+    const [totalYearIterations, setTotalIterations] = React.useState(props.totalIterations);
+
+    const handleChange = (event: SelectChangeEvent<typeof totalYearIterations>) => {
```

Comment:
Just do 'SelectChangeEvent<'number'>' (single quotes because Github comment templating). Expected value from the control is 'number' and 'totalYearIterations' isn't a type... but mind blown that the compiler allowed 'typeof totalYearIterations'.